EventFilter/L1TRawToDigi/python/caloStage1Raw_cfi.py

```diff
@@ -3,13 +3,13 @@
 caloStage1Raw = cms.EDProducer(
     "L1TDigiToRaw",
     Setup = cms.string("stage1::CaloSetup"),
-    InputLabel = cms.InputTag("caloStage1Digis"),
-    TauInputLabel = cms.InputTag("caloStage1Digis", "rlxTaus"),
-    IsoTauInputLabel = cms.InputTag("caloStage1Digis", "isoTaus"),
-    HFBitCountsInputLabel = cms.InputTag("caloStage1Digis", "HFBitCounts"),
-    HFRingSumsInputLabel = cms.InputTag("caloStage1Digis", "HFRingSums"),
-    RegionInputLabel = cms.InputTag("caloStage1Digis", ""),
-    EmCandInputLabel = cms.InputTag("caloStage1Digis", ""),
+    InputLabel = cms.InputTag("simCaloStage1FinalDigis"),
+    TauInputLabel = cms.InputTag("simCaloStage1FinalDigis", "rlxTaus"),
+    IsoTauInputLabel = cms.InputTag("simCaloStage1FinalDigis", "isoTaus"),
+    HFBitCountsInputLabel = cms.InputTag("simCaloStage1FinalDigis", "HFBitCounts"),
+    HFRingSumsInputLabel = cms.InputTag("simCaloStage1FinalDigis", "HFRingSums"),
+    RegionInputLabel = cms.InputTag("simRctDigis", ""),
+    EmCandInputLabel = cms.InputTag("simRctDigis", ""),
```

Comment:
Sorry, no actually, that is a correct correction.  We produce RAW from simulated digis (or the detector does it for us).  So the old version was a mistake.  This one is correct.